crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -804,6 +802,11 @@ impl Starknet {
             return Err(Error::UnsupportedAction { msg: "Block is already aborted".into() });
         }
 
+        let genesis_block = self.blocks.get_by_block_id(&BlockId::Number(0)).unwrap();
```

Comment:
In case of forking, genesis_block won't have number 0

Reply:
Fixed in ebd8339e2576b5f0d94c108d360c95b15544bc58

Reply:
Look at this: https://github.com/0xSpaceShard/starknet-devnet-rs/blob/45646e73767632d9cb7a04bf2cd2e13d89fc59d6/crates/starknet-devnet-core/src/starknet/mod.rs#L193-L196

Reply:
Fixed in 5e17ea91a7f6c4c5f89fa155a4273d0beee1710d